frontends/web/src/utils/markup.test.tsx

```diff
@@ -15,38 +15,39 @@
  */
 
 import 'jest';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
 import { multilineMarkup, SimpleMarkup } from './markup';
 
 describe('SimpleMarkup', () => {
   it('contains a strong element with the text bar', () => {
-    const paragaph = mount(<SimpleMarkup tagName="p" markup="foo <strong>bar</strong> baz" />);
-    expect(paragaph.getDOMNode().nodeName).toEqual('P');
-    expect(paragaph.find('strong').text()).toEqual('bar');
+    const { container } = render(<SimpleMarkup tagName="p" markup="foo <strong>bar</strong> baz" />);
+    // container is a div that wraps whatever element that's being rendered
+    expect(container.firstElementChild?.nodeName).toBe('P');
```

Comment:
Could avoid having `firstElementChild` by using `data-testid` for the tested element (SimpleMarkup). 

But, with the way SimpleMarkup works rn, adding `data-testid` might only increase the code's complexity without adding real value / benefits.